app/src/main/java/org/mozilla/focus/home/HomeTilePlaceholderGenerator.kt

```diff
@@ -17,18 +17,17 @@ import org.mozilla.focus.utils.UrlUtils
 class HomeTilePlaceholderGenerator {
 
     companion object {
-        private val TEXT_SIZE_DP = 52f
+        private val TEXT_SIZE_DP = 22f
         private val DEFAULT_ICON_CHAR = '?'
 
         fun generate(context: Context, url: String?): Bitmap {
             val startingChar = getRepresentativeCharacter(url)
 
             // When this method is called, the view we're setting this bitmap on has no size (maybe
-            // because it doesn't contain an image?) so we use a hard-coded estimated size instead.
-            val width = context.resources.getDimensionPixelSize(R.dimen.home_tile_icon_estimated_width)
-            val height = context.resources.getDimensionPixelSize(R.dimen.home_tile_icon_estimated_height)
+            // because it doesn't contain an image?) so we use a hard-coded size instead.
```

Comment:
nit: this comment isn't useful anymore: can you figure out why? Please respond here and remove it.

If you ever find comments/code that you think are unnecessary, feel free to remove it! Someone will complain if it shouldn't have been removed. :)

Reply:
I think it's because we're simply just giving it a hard-coded size at this point so no need to justify.

Reply:
Yep! 👍